### ISSUE TITLE
[Blocker] It's impossible to install with npm install on Windows.

### ISSUE THREAD

**Issue by Steel5rat** [boardzilla/boardzilla-devtools#10]
I'm trying to run sample boardzilla project on windows, and getting following error during npm install:
![image](https://github.com/boardzilla/boardzilla-devtools/assets/3667519/304c17b9-63be-4f67-aa81-cdb01938d451)

I suppose it's because it should use `.zip` extension for windows package. I believe such customization currently isn't supported in `go-npm` lib that you're using. I've created a PR https://github.com/joshbuddy/go-npm/pull/1 to introduce it, so it would be nice to merge it, update go-npm lib, and change [devtools goBinary.url](https://github.com/boardzilla/boardzilla-devtools/blob/5e0a29a1b91ced8b02049de5af9c2a75086c7e82/package.json#L25) to have a proper link for Windows.

**Comment by david-torres** [comment]
Same issue though it seems to be because there's no build for windows_amd64

**Comment by goodsky** [comment]
I found two additional issues:
1. When installing using `npm` on Windows with a recent version of Node (18+) it will fail to detect the .bin directory. This PR addresses that issue: https://github.com/joshbuddy/go-npm/pull/3
2. The `unzipper` package is apparently broken with Node 20+ and causes the bz.exe file to be corrupt. This PR moves to use `unzip-stream` instead: https://github.com/joshbuddy/go-npm/pull/4

**Comment by joshbuddy** [comment]
Okay, I've got some news. Given the devtools are already just a go binary, I've decided to skip the middleman and ship that directly. If you could check out this link, and tell me if any of these methods are acceptable to you

https://docs.boardzilla.io/introduction/getting-started#installing-the-devtools

I could also enable `winget`, but want to hear from you first to see if that effort would be useful.

**Comment by goodsky** [comment]
That sounds reasonable to me (and more sustainable than fiddling with `go-npm` 🙂).

Of the options in the documentation, I will opt to build from source for now. At some point, I would expect a Windows installer to download. (In fact, I think having an installer may be a prerequisite for winget?). But I understand that being a low priority at the moment as long as we're unblocked.

**Comment by joshbuddy** [comment]
Yeah, I think I'd like to support installers for every major platform, including Windows. Just need a couple of days to get it all ironed out. Glad you're unblocked

**Comment by joshbuddy** [comment]
Okay, did a bunch more testing and fixes and Windows should be working now. Please let know if everything is good now
